doc/arkode/guide/source/ARKodeSPRKStorage.rst

```diff
@@ -0,0 +1,177 @@
+.. ----------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2023, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   ----------------------------------------------------------------
+
+.. _SPRKStorage:
+
+==============================
+SPRK Method Storage Structure
+==============================
+
+.. _SPRKStorage.id:
+
+Method identifiers
```

Comment:
Should we remove them from Butcher.rst instead of removing them from here since they are separate from the Butcher table structure?

Reply:
I think leaving these in Butcher.rst make sense so all the method tables are listed in one place (even if they correspond to different data structures for storing the table). They're already under a "Symplectic" section and some introductory text could be add at the start of the section to note the different data structure used in this case. It might also be better to name this section "SPRK Tables" since to, as Dan mentioned somewhere else, distinguish from other symplectic methods with a different table structure.